cli.py

```diff
@@ -54,32 +54,38 @@ def submit_to_biosamples(self, service: BioSamples):
                 response = service.send_sample(request)
                 if 'accession' in response:
                     sample.add_accession('BioSamples', response['accession'])
-                    sample.attributes['sample_accession'] = response['accession']
             except Exception as error:
                 error_msg = f'BioSamples Error: {error}'
                 sample.add_error('sample_accession', error_msg)
 
     def submit_to_biostudies(self, service: BioStudies):
         for study in self.excel.data.get_entities("study"):
-            bio_study_submission = BioStudyConverter.convert_study(study)
-            accession = service.send_submission(bio_study_submission)
-            study.add_accession('BioStudies', accession)
-            study.attributes['bio_study_accession'] = accession
+            try:
+                bio_study_submission = BioStudyConverter.convert_study(study)
+                accession = service.send_submission(bio_study_submission)
+                study.add_accession('BioStudies', accession)
+            except Exception as error:
+                error_msg = f'BioStudies Error: {error}'
```

Comment:
These 2 lines are the same as line 79-80. We could extract it to a method.
You might use it at line 222, too.

Reply:
I agree, this is quite redundant.
I think the CLI might change quite drastically if/when we do the split-apart refactor.
Let's consider methods for safe error trapping and reporting then.

Reply:
I agree, this is quite redundant.
I think the CLI might change quite drastically if/when we do the split-apart refactor.
Let's consider methods for safe error trapping and reporting then.